_posts/en/2019-10-21-schnorr-taproot-workshop.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Bitcoin Optech Schnorr Taproot Workshop
+permalink: /en/schorr-taproot-workshop/
+name: 2019-10-21-schnorr-taproot-workshop
+type: posts
+layout: post
+lang: en
+slug: 2019-10-21-schnorr-taproot-workshop
+
+excerpt: >
+  A self-study course for learning about the schnorr/taproot softfork proposal.
+
+auto_id: false
+---
+{% include references.md %}
+
+In September 2019, Bitcoin Optech organized two workshops in San Francisco and
+New York on the schnorr/taproot softfork proposals. The aims of the workshop
+were to:
+
+1. share current thinking in the open source community on the proposals;
+2. give engineers a chance to work with the new technology through interactive
+   jupyter notebooks;
+3. help engineers take part in the community feedback process.
+
+This blog post contains all of the videos, slides and jupyter notebooks from
+those workshops, so engineers at home can learn about these exciting new
+technologies.
+
+The workshop is split into 4 sections:
+
+1. [Preparation and basic math](#preparation-and-basic-math) - shows how to set
+  up the jupyter notebook environment, gives a refresher on basic elliptic
+  curve math, and introduces tagged hashes.
+2. [Schnorr signatures and MuSig](#schnorr-signatures-and-musig) - describes
+  the bip-schnorr signature scheme, and how to use MuSig to aggregate multiple
```

Comment:
nit: remove the commit here since the second clause isn't a proper sentence/independent clause.